ledger/sandbox-on-x/reference.conf

```diff
@@ -90,9 +110,12 @@ ledger {
 
     # The optional specifies the directory where to save the output of the Daml
     # scenario profiler. The profiler is disabled if the option is empty.
+    # TODO enterprise feature, should not be part of public code
```

Comment:
This is necessary for the sdk CI tests. There is no mechanism at this moment to separate the enterprise from community functionality of the sdk that is already there